src/components/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro'
+
+function ProductCard({ name, description, image }) {
+  return (
+    <ProductCardWrapper>
+      <ProductImg src={image} alt={`${name} image`} />
+      <InfoWrapper>
+        <h2>{name}</h2>
+        <p>{description}</p>
+      </InfoWrapper>
+    </ProductCardWrapper>
```

Comment:
How about this? You are already in the `ProductCard` component.
```suggestion
    <Wrapper>
      <Image src={image} alt={`${name} image`} />
      <InfoWrapper>
        <h2>{name}</h2>
        <p>{description}</p>
      </InfoWrapper>
    </Wrapper>
```